packages/react-i18n/src/i18n.ts

```diff
@@ -500,6 +510,27 @@ export class I18n {
     });
   }
 
+  private humanizeFutureDate(date: Date, options?: Intl.DateTimeFormatOptions) {
+    const time = this.formatDate(date, {
+      ...options,
+      hour: 'numeric',
+      minute: '2-digit',
+    }).toLocaleLowerCase();
+
+    if (isToday(date)) {
+      return time;
+    }
+
+    if (isTomorrow(date)) {
+      return this.translate('date.humanize.tomorrow', {time});
+    }
+
+    return this.formatDate(date, {
+      ...options,
```

Comment:
Same thing here

Reply:
See [this comment](https://github.com/Shopify/quilt/pull/1391#discussion_r413034933)